examples/sftp-listener/sftp_listener.out

```diff
@@ -0,0 +1,4 @@
+bal run sftp.bal
```

Comment:
```suggestion
bal run sftp_listener.bal
```

Reply:
Changed all the output content using the file names given in the code itself.